--- origsrc/libtheora-1.0beta3/Makefile.am	2008-04-16 15:25:55.000000000 -0500
+++ src/libtheora-1.0beta3/Makefile.am	2008-05-23 08:52:37.843750000 -0500
@@ -1,6 +1,7 @@
 ## Process this file with automake to produce Makefile.in
 
 AUTOMAKE_OPTIONS = foreign 1.6 dist-zip dist-bzip2
+ACLOCAL_AMFLAGS = -I m4
 
 if THEORA_ENABLE_EXAMPLES
 EXAMPLES_DIR = examples
--- origsrc/libtheora-1.0beta3/configure.ac	2008-04-16 15:25:55.000000000 -0500
+++ src/libtheora-1.0beta3/configure.ac	2008-05-23 08:30:21.453125000 -0500
@@ -43,7 +43,7 @@
 AC_SUBST(THENC_LIB_AGE)
 
 dnl Extra linker options (for version script)
-THEORA_LDFLAGS=""
+THEORA_LDFLAGS="-no-undefined"
 
 dnl --------------------------------------------------  
 dnl Check for programs
@@ -374,10 +374,10 @@
 AC_SUBST(GETOPT_OBJS)
 
 if test x$HAVE_SDL = xyes -a x$HAVE_OSS = xyes -a x$HAVE_VORBIS = xyes; then
-  BUILDABLE_EXAMPLES="$BUILDABLE_EXAMPLES player_example"
+  BUILDABLE_EXAMPLES="$BUILDABLE_EXAMPLES player_example\$(EXEEXT)"
 fi
 if test x$HAVE_PNG = xyes; then
-  BUILDABLE_EXAMPLES="$BUILDABLE_EXAMPLES png2theora"
+  BUILDABLE_EXAMPLES="$BUILDABLE_EXAMPLES png2theora\$(EXEEXT)"
 fi
 AC_SUBST(BUILDABLE_EXAMPLES)
 
--- origsrc/libtheora-1.0beta3/lib/Makefile.am	2008-04-16 15:25:54.000000000 -0500
+++ src/libtheora-1.0beta3/lib/Makefile.am	2008-05-23 09:43:22.515625000 -0500
@@ -144,6 +144,7 @@
 libtheoraenc_la_LDFLAGS = \
   -version-info @THENC_LIB_CURRENT@:@THENC_LIB_REVISION@:@THENC_LIB_AGE@ \
   @THEORAENC_LDFLAGS@ $(OGG_LIBS)
+libtheoraenc_la_LIBADD = libtheoradec.la
 
 libtheora_la_SOURCES = \
   cpu.c \
